Sources/TuistLoader/SwiftPackageManager/PackageInfoMapper.swift

```diff
@@ -843,14 +843,26 @@ extension ProjectDescription.ResourceFileElements {
         // Add default resources path if necessary
         // They are handled like a `.process` rule
         if sources == nil {
-            resourceFileElements += try defaultResourcePaths(from: path)
+            // Already included resources should not be added as default resource
+            let excludedPaths: Set<AbsolutePath> = Set(
+                resourceFileElements.map {
+                    switch $0 {
+                    case let .folderReference(path: path, _, _):
+                        AbsolutePath(stringLiteral: path.pathString)
+                    case let .glob(pattern: path, _, _, _):
+                        // TODO: What's the correct way to remove the ** pattern from the string?
+                        AbsolutePath(stringLiteral: path.pathString).parentDirectory
+                    }
+                }
+            )
+            resourceFileElements += try defaultResourcePaths(from: path, excludingPaths: excludedPaths)
                 .compactMap { try handleProcessResource(resourceAbsolutePath: $0) }
         }
 
         // Check for empty resource files
         guard !resourceFileElements.isEmpty else { return nil }
 
-        return .resources(resourceFileElements)
+        return .resources(resourceFileElements.spm_uniqueElements())
```

Comment:
```suggestion
        return .resources(resourceFileElements.uniqued())
```

You should be able to use this instead.